### ISSUE TITLE
new feature

### ISSUE THREAD

**Issue by prosecurity** [gotr00t0day/AUT0-REC0N#1]
is it possible to provide a domain to enumerate such as find subdomains and then run directory scan on all the sub domains found and also maybe run nmap to find open ports on all subdomains

**Comment by Knowledge-Wisdom-Understanding** [comment]
@prosecurity  Yes I'm sure that is quite possible. I started porting this bash script to python3 ,
[python3 version](https://github.com/gotr00t0day/RECON)
I will see if I can add this option to the python3 version.
I'm no longer actively maintaining the bash script so I would recommend using the python3 version for now, however, If you would like to submit a pull request with this feature, I'll gladly review it and add it to the bash version if it works effectively. I will look into adding this feature to the python3 version.

**Comment by Knowledge-Wisdom-Understanding** [comment]
Also, @prosecurity  Check out `https://github.com/Screetsec/Sudomy` . This tool sounds like it does what you're looking for. I may incorporate this tool into my recon scripts in the future as an option. Stay tuned.
